packages/quantic/force-app/main/default/lwc/quanticFeedback/quanticFeedback.js

```diff
@@ -6,9 +6,9 @@ import {LightningElement, api} from 'lwc';
 
 /**
  * The `QuanticFeedback` component displays a question and two buttons for giving positive or negative feedback.
- * @fires CustomEvent#like
- * @fires CustomEvent#dislike
- * @fires CustomEvent#pressExplainWhy
+ * @fires CustomEvent#quantic__like
+ * @fires CustomEvent#quantic__dislike
+ * @fires CustomEvent#quantic__pressExplainWhy
```

Comment:
The name convention that currently be used is to have all letters in lower case.

Reply:
will update that